src/auth/auth.module.ts

```diff
@@ -0,0 +1,18 @@
+import { Module } from '@nestjs/common';
+import { PassportModule } from '@nestjs/passport';
+import { UsersModule } from 'src/users/users.module';
+import { AuthService } from './auth.service';
+import { LocalStrategy } from './local.strategy';
+import { LocalSerializer } from './local.serializer';
+import { MongooseModule } from '@nestjs/mongoose';
+import { User, UserSchema } from 'src/schemas/user.schemas';
+
+@Module({
+  imports: [
+    MongooseModule.forFeature([{ name: User.name, schema: UserSchema }]),
+    UsersModule,
+    PassportModule,
```

Comment:
```suggestion
    PassportModule.register({ session: true })
```
세션 로그인이니깐 세션 추가해줘야 해요. 강의에도 나와있었을거에요. 기억은 안나네요.